### ISSUE TITLE
Nested components (missing feature)

### ISSUE THREAD

**Issue by Mica4DEV** [knockout/knockout#1562]
About the new component feature in ko 3.2:
There is no possibility to have nested components in html, and to keep child components.

Example, two custom components:

``` html
<panelgroup>
    <panel></panel>
    <panel></panel>
</panelgroup>
```

If panelgroup is a component, it will put the template within itself (and destroy its children).
We need to have a way to keep existing children, and be able to access them in order to put it in a specific place of the parent component template.

Example, with that template
**----- panelgrouptemplate -----**

``` html
     <div class="panelgroup">
         <div class="child-counter" data-bind="text:count"></div>
         <div>{{children-components}}</div>
     </div>
```

It would end up with (considering the previous code):

``` html
<panelgroup>
     <div class="panelgroup">
          <div class="child-counter" data-bind="text:count"></div>
         <div>
               <panel></panel>
               <panel></panel>
         </div>
    </div>
</panelgroup>
```

And of course `<panel>` could be a component that would transform itself, as every knockout template would inner auto "applyBindingsToDescendants" would run.

If I'm not mistaking, we're unable to do that with the current 3.2 version.

**Comment by lyt23** [comment]
wait the feature，we need it

**Comment by mafre** [comment]
This would be very helpful! Any plans on releasing this feature anytime soon?

**Comment by mbest** [comment]
@mafre It's included in 3.3.0 (See #1463).
